docs/common-problems.asciidoc

```diff
@@ -3,8 +3,78 @@
 
 This section describes common problems you might encounter with APM Server.
 
+* <<no-data-indexed>>
+* <<bad-request>>
+* <<unauthorized>>
+* <<forbidden>>
+* <<request-too-large>>
+* <<rate-limited>>
 * <<queue-full>>
 * <<request-timed-out>>
+* <<ssl-client-fails>>
+
+[[no-data-indexed]]
+[float]
+=== No data is indexed
+If no data shows up in Elasticsearch or Kibana, first check that the APM components are properly connected.
```

Comment:
Index implies just elasticsearch to me, I think remove kibana here.  The UI in kibana already has good built-in help when there is no data.
```
No services with data in the selected time range.
Oops! You should try another time range. If that's no good, there's always the documentation.
```